app/code/Magento/CustomerGraphQl/etc/schema.graphqls

```diff
@@ -51,3 +51,13 @@ type CustomerAddressRegion @doc(description: "CustomerAddressRegion defines the
     region_id: Int @doc(description: "Uniquely identifies the region")
 }
 
+type Mutation {
```

Comment:
Also, need to cover functionality with API-functional tests
Now we have a lot of examples and tests writing is very simple